packages/cozy-client/src/models/document/locales/index.js

```diff
@@ -26,7 +26,14 @@ const getBoundT = lang => {
       ...opts,
       smart_count: opts?.smart_count || 1
     }
-    const emojiCountry = getEmojiByCountry(opts?.country, t)
+    const country = opts?.country
+
+    let emojiCountry
+    if (country === 'stranger') {
+      emojiCountry = t('country.stranger')
```

Comment:
It looks weird to have a locale in the `emojiCountry`. Why note something like this?
```js
    const emojiCountry = country !== 'stranger' ? getEmojiByCountry(country) : null
    const countryLabel = country === 'stranger' ? t('country.stranger') : label

    return emojiCountry
      ? `${t(countryLabel, newOpts)} ${emojiCountry}`
      : t(countryLabel, newOpts)
```

Reply:
Indeed, it looks better 👍

Reply:
I've set it as below so as not to change the existing return.
```js
const emojiCountry = country !== 'stranger' ? getEmojiByCountry(country) : null
const strangerLabel = country === 'stranger' ? t('country.stranger') : null

return emojiCountry || strangerLabel
  ? `${t(label, newOpts)} ${emojiCountry || strangerLabel}`
  : t(label, newOpts)
```